blog/tests/test_blog_article_page.py

```diff
@@ -107,3 +106,32 @@ def test_that_article_with_no_recommended_articles_should_be_valid(self):
         blog_article_page.save()
 
         self.assertEqual(blog_article_page.recommended_articles, StreamValue(articles_block, []))
+
+    def test_that_article_intro_should_be_a_part_of_first_paragraph_under_fixed_amount_of_characters(self):
+        paragraph_string = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam laoreet venenatis enim, non luctus nisi finibus ut. Mauris porta eleifend massa, nec maximus lacus luctus a. Aenean libero felis, placerat non malesuada a, maximus id erat. Nulla ut purus elementum, auctor orci eget, facilisis est."
+        expected_string = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam laoreet venenatis enim, non luctus nisi finibus ut. Mauris porta eleifend massa, nec maximus lacus luctus a. Aenean libero felis, placerat non malesuada a, maximus id erat. Nulla ut"
```

Comment:
I was wondering if we shouldn't end this string with 3 dots "..."? It might cut sentence in the middle and we should point that this is not the end of it. What do you think?

Reply:
The three dots `[...]` were originally added at the front end (in the index page template), so I left it there. Should I move it here, to the backend?